packages/vanilla/src/sass/layout/_container.scss

```diff
@@ -1,8 +1,8 @@
 @use './../tokens/breakpoints' as *;
 
 .cbp-fluid-container {
-  padding-left: var(--cbp-space-4x);
-  padding-right: var(--cbp-space-4x);
+  padding-left: var(--cbp-space-5x);
+  padding-right: var(--cbp-space-5x);
 
   @media (min-width: $cbp-breakpoint-md) and (max-width: $cbp-breakpoint-xl) {
```

Comment:
I know this line hasn't actually changed, but you don't need the upper limiter because you have another media query after it that will take precedence, directly overriding these rules.